### DISCUSSION TITLE
参考文献顺序问题

### DISCUSSION THREAD

**Discussion by shawchoy** [note286/xduts#163] [Q&A] [answered]
表格标题中引用的参考文献会出现在正文（从第一章绪论开始）引用的参考文献之前
我想这可能是因为把表格索引也当成正文所以参考文献排在正文引用之前了

**Comment by sikouhjw** [comment]
解决方案：`\caption[不出现 \cite]{出现 \cite}`。为什么要关闭 discussion？

**Reply by shawchoy** [reply]
关闭是因为我这个问题出现在overleaf编译时，我在本地编译不会出现这个问题

**Comment by note286** [comment, marked as answer]
根据你的描述，你应该是使用了`\xdusetup{style/bib-backend=bibtex}`切换到了`bibtex`，你可以添加

```latex
\usepackage{notoccite}
```

来规避这个问题，正文中无需任何修改。

另外，如果你使用默认的`biblatex`，则不会有这种情况出现。

**Reply by shawchoy** [reply]
这个方法有效，谢谢

**Comment by shawchoy** [comment]


**Reply by note286** [reply]
请不要讨论与当前主题无关的问题。
